Package.swift

```diff
@@ -129,6 +129,10 @@ let package = Package(
       name: "FirebaseRemoteConfigSwift",
       targets: ["FirebaseRemoteConfigSwift"]
     ),
+    .library(
+      name: "FirebaseSessions",
+      targets: ["FirebaseSessions"]
+    ),
     .library(
       name: "FirebaseStorage",
       targets: ["FirebaseStorage"]
```

Comment:
Line 167 below: GoogleUtilities 7.10.0 is now required.

Reply:
Ooh good catch - updated.

On a similar vein, I've been trying to figure out a way to get it to build for the lowest iOS / watchOS version we support instead of the highest.